src/LogFileReader.cpp

```diff
@@ -74,12 +81,18 @@ bool LogFileReader::readAll(const QString& fileName)
 // File input is a csv file in the format of hh:mm:ss:zzz, voltage, current
 bool LogFileReader::parseLine(const QString& line, BatteryData& batteryData) const
 {
-    // TODO implement this first
-
-    // This is here to the compiler happy. Otherwise the compile
-    // will have an error warning about an unused variable. Remove this
-    // when you use it.
-    Q_UNUSED(line);
-    Q_UNUSED(batteryData);
-    return true;
+    TEMP_TIME = line.section(",",0,0);
+    QTime bdt = QTime::fromString(TEMP_TIME, TIME_FORMAT);
+    double bdv= line.section(", ",1,1).toDouble();
+    double bdc = line.section(", ",2,2).toDouble();
```

Comment:
Generally it's poor practice to abbreviate names, have a longer descriptive name is much better than having a short one for readability reasons. Just expanding them would be pretty good, so `bdt --> batteryDataTime` and so forth.